server/src/lib/routes/secondfactor/u2f/register/post.ts

```diff
@@ -10,12 +9,16 @@ import { ServerVariables } from "../../../../ServerVariables";
 import { AuthenticationSessionHandler } from "../../../../AuthenticationSessionHandler";
 import UserMessages = require("../../../../../../../shared/UserMessages");
 import { AuthenticationSession } from "../../../../../../types/AuthenticationSession";
+import GetHeader from "../../../../utils/GetHeader";
+import * as Constants from "../../../../../../../shared/constants";
 
 
 export default function (vars: ServerVariables) {
   function handler(req: express.Request, res: express.Response): BluebirdPromise<void> {
     let authSession: AuthenticationSession;
-    const appid = u2f_common.extract_app_id(req);
```

Comment:
[BREAKING] This was using the Host header but now uses the X-Forwarded-Host header.